lib/popcorn/api/wasm.ex

```diff
@@ -308,9 +309,8 @@ defmodule Popcorn.Wasm do
   end
 
   defp run_js_fn(code) do
-    with {:ok, ref} <- :emscripten.run_remote_object_fn_script(code, main_thread: true) do
-      {:ok, %RemoteObject{ref: ref}}
-    end
+    {:ok, ref} = :emscripten.run_remote_object_fn_script(code, main_thread: true)
+    {:ok, %RemoteObject{ref: ref}}
```

Comment:
Why we change that?

Reply:
Because `run_remote_object_fn_script` never returns anything else. But I should be consistent and return plain `RemoteObject`

Reply:
Ah, right. We raise on errors there. `get_remote_object` works in a different way though.

Reply:
Yup, it could be more consistent